packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_navigation.spec.ts

```diff
@@ -1,5 +1,5 @@
 // @ts-ignore
-context('multi-domain navigation', { experimentalMultiDomain: true }, () => {
+context('multi-domain navigation', { experimentalMultiDomain: true, experimentalSessionSupport: true }, () => {
```

Comment:
I don't think the flag is needed here since this should be turned on globally for driver tests
```suggestion
context('multi-domain navigation', { experimentalSessionSupport: true }, () => {
```

Reply:
Removed. [a81f3f4](https://github.com/cypress-io/cypress/pull/20571/commits/a81f3f477d5badd6250e651f8fffe250df8dc69c)